app/helpers/budget_warning.rb

```diff
@@ -13,20 +13,18 @@ def initialize
     end
 
     def go_live?
-      (Date.today >= date) ? true : false
+      Time.zone.today >= date ? true : false
     end
   end
 
   def display_banner_warning?
-    BudgetWarning::Announcement.new.go_live?
+    BudgetWarning::Announcement.new.go_live? && whitelisted?
   end
   module_function :display_banner_warning?
 
   def whitelisted?
-    whitelist = ['/en/tools/redundancy-pay-calculator/',
-                 '/cy/tools/cyfrifiannell-tal-diswyddo/']
+    whitelist = YAML.load_file('config/budget_warning_tools.yml')
```

Comment:
we should not trigger some IO, every time it is invoked. We should be able to cache that.